lib/protocol/SqlString.js

```diff
@@ -1,10 +1,12 @@
 var SqlString = exports;
 
-SqlString.escapeId = function (val, forbidQualified) {
+SqlString.escapeId = function escapeId(val, forbidQualified) {
   if (Array.isArray(val)) {
-    return val.map(function(v) {
-      return SqlString.escapeId(v, forbidQualified);
-    }).join(', ');
+    var sql = '';
+    for (var i = 0; i < val.length; i++) {
+      sql += (i ? ', ' : '') + escapeId(val[i], forbidQualified);
```

Comment:
The change from `SqlString.escapeId` to just `escapeId` will affect people who are using AOP around this function, since we have been calling it as a method. Just noting this if you wonder why that part is reverted in the merge of your pull request.